src/store.ts

```diff
@@ -326,6 +331,7 @@ export const initializeFirestoreListeners = (onLoad: () => void): (() => void) =
       const plan = getFirstPlan(data);
       store.commit('setPlans', data.plans);
       store.commit('setCurrentPlan', plan);
+      // store.commit('setCollections', data.collections); Note: toggle this on and off to save collections progress after refresh
```

Comment:
Thank you for haring this here and in the main PR information section. Tiny quality of life improvement could be to add another `//` before the `Note: ` so you don't have to add that when toggling, but really just nitpicking.